webpack.dev.js

```diff
@@ -0,0 +1,18 @@
+const webpack = require('webpack');
+const merge = require('webpack-merge');
+const CleanTerminalPlugin = require('clean-terminal-webpack-plugin');
+
+const common = require('./webpack.common.js');
+
+
+module.exports = merge(common, {
+  devtool: 'inline-source-map',
+  plugins: [
+    new CleanTerminalPlugin(),
+    new webpack.DefinePlugin({
+      'process.env': {
+        NODE_ENV: JSON.stringify('development'),
```

Comment:
why do we need JSON.stringify here? I think `"'development'"` or `\""development"\"` might work too